bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -1,488 +1,177 @@
-    depends_on("ncurses")
-    depends_on("python@2.6:", when="+python", type=("build", "link", "run"))
-    depends_on("py-pytest", when="+python+tests")
-    # Numpy is required for Vector.as_numpy()
-    depends_on("py-numpy", when="+python", type=("build", "run"))
-    depends_on("py-cython", when="+rx3d", type="build")
-    depends_on("py-pytest-cov", when="+tests@8:")
-    # TODO: newer spack doesn't propogate python package as dependency
-    # (as PYTHONPATH) during build time
-    depends_on("py-sympy@1.3:", type=("build", "run"))
-    # Next two needed in versions containing neuronsimulator/nrn#2235.
-    depends_on("py-pip", type=("build"), when="@9:")
-    depends_on("py-packaging", type=("run"), when="@9:")
-
-    # dependency on coreneuron via submodule
-    depends_on("coreneuron+legacy-unit~caliper", when="@:8.99+coreneuron+legacy-unit~caliper")
-    depends_on("coreneuron~legacy-unit~caliper", when="@:8.99+coreneuron~legacy-unit~caliper")
-    depends_on("coreneuron+legacy-unit+caliper", when="@:8.99+coreneuron+legacy-unit+caliper")
-    depends_on("coreneuron~legacy-unit+caliper", when="@:8.99+coreneuron~legacy-unit+caliper")
-    conflicts("coreneuron", when="@9:")
-
-    # dependencies from coreneuron package
-    depends_on("python", type=("build", "run"))
-    depends_on("boost", when="@8.99:+tests+coreneuron")
-    depends_on("cuda", when="@8.99:+gpu")
-    depends_on("libsonata-report", when="@8.99:+report+coreneuron")
+    # used during development by core team
+    variant(
+        "sanitizers",
+        default="None",
+        description="Enable runtime sanitizers",
+        multi=True,
+        values=("None", "address", "leak", "undefined"),
+    )
 
-    # for coreneuron: some basic conflicts
-    gpu_compiler_message = "neuron: for gpu build use %pgi or %nvhpc"
-    conflicts("%gcc", when="+gpu", msg=gpu_compiler_message)
-    conflicts("%intel", when="+gpu", msg=gpu_compiler_message)
+    # standard deployment uses submodule to avoid compatibility issues
+    depends_on("nmodl", when="+coreneuron")
+    depends_on("libsonata-report", when="+report+coreneuron")
 
-    # ==============================================
-    # ==== CMake build system related functions ====
-    # ==============================================
     def cmake_args(self):
-        def cmake_enable_option(spec_requiremement):
-            value = "TRUE" if spec_requiremement in self.spec else "FALSE"
-            cmake_name = spec_requiremement[1:].upper().replace("-", "_")
-            return "-DNRN_ENABLE_" + cmake_name + ":BOOL=" + value
-
-        args = [
-            cmake_enable_option(variant)
-            for variant in [
-                "+interviews",
-                "+python",
-                "+memacs",
-                "+rx3d",
-                "+coreneuron",
-                "+tests",
-            ]
-        ]
-        if self.spec.satisfies("+tests"):
+        args = super(Neuron, self).cmake_args()
```

Comment:
```suggestion
        args = super().cmake_args()
```
Python 3 only by now.

Reply:
done in ff51c40c6b187e9c98e804b1bfa3c3a58e1af6a8